src/TriggerBinding/SqlTableChangeMonitor.cs

```diff
@@ -810,6 +815,50 @@ private SqlCommand BuildGetChangesCommand(SqlConnection connection, SqlTransacti
             return new SqlCommand(getChangesQuery, connection, transaction);
         }
 
+        /// <summary>
+        /// Returns the number of changes(rows) on the user's table that are actively locked by other leases.
+        /// </summary>
+        /// <param name="connection">The connection to add to the SqlCommand</param>
+        /// <param name="transaction">The transaction to add to the SqlCommand</param>
+        /// <returns>The number of rows locked by leases</returns>
+        private async Task<int> GetLeaseLockedRowCount(SqlConnection connection, SqlTransaction transaction)
+        {
+            string leasesTableJoinCondition = string.Join(" AND ", this._primaryKeyColumns.Select(col => $"c.{col.name.AsBracketQuotedString()} = l.{col.name.AsBracketQuotedString()}"));
+            int leaseLockedRowsCount = 0;
+            long getLockedRowCountDurationMs = 0L;
+            // Get the count of changes from CHANGETABLE that meet the following criteria:
+            // * Not Null LeaseExpirationTime AND
+            // * LeaseExpirationTime > Current Time
+            string getLeaseLockedrowCountQuery = $@"
+                {AppLockStatements}
+
+                DECLARE @last_sync_version bigint;
+                SELECT @last_sync_version = LastSyncVersion
+                FROM {GlobalStateTableName}
+                WHERE UserFunctionID = '{this._userFunctionId}' AND UserTableID = {this._userTableId};
+
+                SELECT COUNT(*)
+                FROM CHANGETABLE(CHANGES {this._userTable.BracketQuotedFullName}, @last_sync_version) AS c
+                LEFT OUTER JOIN {this._leasesTableName} AS l ON {leasesTableJoinCondition}
```

Comment:
If leases table always guaranteed to be quoted here? If so let's update this var to be `_bracketedLeasesTableName`. And if not, then make sure you do that for users who have custom lease table names.

@lucyzhang929 to help verify this

Reply:
I've debugged this and saw the leasesTableName is bracketed. But I can use the _bracketedLeasesTableName just to be sure.

Reply:
Yes we format the leasesTableName here: https://github.com/Azure/azure-functions-sql-extension/blob/04f64be48d0660471385be1d1407da71f701d2f6/src/TriggerBinding/SqlTriggerUtils.cs#L126.

Reply:
Thanks @lucyzhang929! Am going to leave it as is.

Reply:
I think what Charles' comment meant is to update the variable name `_leasesTableName` to `_bracketedLeasesTableName` so it is clear that the name is correctly formatted.

Reply:
update _leasesTableName  to _bracketedLeasesTableName  everywhere? I can do it in a separate PR, don't want to do that change here.

Reply:
Yup, that sounds good to me. Thanks for looking into it!